overlays/wwroot/persistent/ignition.json.ww

```diff
@@ -0,0 +1 @@
+{{ IgnitionJson }}
```

Comment:
I don't think this needs to be in a bespoke `wwroot` overlay any more. It can just be pulled from the ignition overlay.